app/templates/direct-award/view-project.html

```diff
@@ -36,6 +36,8 @@
   {% endwith %}
 {% endblock %}
 
+{% block body_classes %}direct-award-project-overview-page{% endblock %}
+
```

Comment:
Ahh, interesting, what does this block do, how does it work?

Reply:
this block adds a class to the body tag, have a look at this template:
https://github.com/alphagov/govuk_template/blob/master/source/views/layouts/govuk_template.html.erb#L38

I needed this as the flash message which I was trying to target was inside the body not the content block. As I did not want the change to affect every flash message, I had to add a specific class to keep the scope to that page. 

Hope this makes sense. :)

Reply:
Is there really no more general case that needs tackling here?

For example, maybe: - other pages that have headings? Other pages that have task lists?

It just seems like there isn't / shouldn't be much that's special about this page. All pages may have flash messages.

Reply:
Hey george - I was thinking of doing something more generic but didn't because of the following reasons:
- Removing the margin-bottom completely might affect pages where there isn't a header at the top.
- There may be already specific css to handle the issue on other pages.
- Not sure how we would apply specific CSS for only when headers elements are after the flash message.
- A lot of effort for a small styling issue.

Reply:
Hey George, 

Looked through the apps, the margin-bottom isn't really needed anywhere apart from the homepage.

I will therefore probably remove it from the template and add some custom margin for the homepage.

https://github.com/alphagov/digitalmarketplace-frontend-toolkit/commit/3d2af0544db2c0d77c0788d128d49ed4adcff751

![screen shot 2018-04-20 at 16 12 41](https://user-images.githubusercontent.com/4599889/39059430-5ee0b31e-44b6-11e8-8538-45ee5eef7547.png)

Reply:
Sounds sensible. The wins for CSS simplification work are generally small wins eh :-)